linux_os/guide/services/ssh/ssh_server/sshd_use_approved_ciphers/rule.yml

```diff
@@ -31,8 +44,16 @@ description: |-
     {{{ weblink(link="http://csrc.nist.gov/groups/STM/cmvp/documents/140-1/140sp/140sp2630.pdf") }}}
     {{% endif %}}
 {{% endif %}}
+{{% endif %}}
 
 rationale: |-
+{{% if product == "sle12" %}}
+    Unapproved mechanisms that are used for authentication to the cryptographic module are not verified and therefore
+    cannot be relied upon to provide confidentiality or integrity, and system data may be compromised.
+    <br />
+    Operating systems utilizing encryption are required to use DoD-approved ciphers to protect the confidentiality
+    of SSH remote connections.
+{{% else %}}
```

Comment:
Nack on this change. FIPS is mandated by Federal Law. This change is not needed or desired.

Reply:
If DoD is providing their own approved ciphers outside of our national standards (NSA and NIST), we could create a rule similar to ``sshd_use_dod_approved_ciphers``. Documentation/sources would have to be provided showing what cipher suites should go into such a rule. But it may be an alternative path forward.

Reply:
@shawndwells Rather than create a separate rule, a variable could be used instead, but really, DISA should follow the FIPS guidelines to be 800-53 compliant.

Reply:
I've reverted the commit. Not sure yet how we'll handle the difference with our STIG, but I don't want to block this pull request any longer.

I'm keeping the conversation unresolved, so I can find it again when I come back to it.